swagger-ui/swagger-ui.js

```diff
@@ -42,6 +42,18 @@ this["Handlebars"]["templates"]["content_type"] = Handlebars.template({"1":funct
 },"useData":true});
 'use strict';
 
+var replaceResponse = function(a, b, element) {
+  if (!element) element = document.body;
+  var nodes = element.childNodes;
+  for (var n=0; n<(nodes.length ? nodes.length : 0); n++) {
+    if (nodes[n].nodeType == Node.TEXT_NODE) {
+      var reg = new RegExp(a, 'gi');
```

Comment:
If using a regex, declare it in an outer scope since building a regex is relatively expensive